Library/Formula/offline-imap.rb

```diff
@@ -14,6 +12,10 @@ def install
     bin.install_symlink libexec+"offlineimap.py" => "offlineimap"
   end
 
+  test do
+    system bin/"offlineimap", "--version"
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!